zookeeper-server/src/test/java/org/apache/zookeeper/common/X509TestContext.java

```diff
@@ -214,6 +219,28 @@ private File getTrustStorePemFile() throws IOException {
         return trustStorePemFile;
     }
 
+    private File getTrustStorePkcs12File() throws IOException {
+        if (trustStorePkcs12File == null) {
+            try {
+                File trustStorePkcs12File = File.createTempFile(
+                    TRUST_STORE_PREFIX, KeyStoreFileType.PKCS12.getDefaultFileExtension(), tempDir);
+                trustStorePkcs12File.deleteOnExit();
+                final FileOutputStream trustStoreOutputStream = new FileOutputStream(trustStorePkcs12File);
```

Comment:
Use try-with-resources (change all the file)

Reply:
Done in this file